src/components/Eth2Articles.js

```diff
@@ -31,49 +32,49 @@ const RightColumn = styled.div`
 
 const dannyArticles = [
```

Comment:
FYI since these are updated quite regularly (& the articles themselves are in English) I'm thinking we don't end up translating these. I'll revert these changes.